### DISCUSSION TITLE
Scintilla code part of the 2000 lines of well commented code?

### DISCUSSION THREAD

**Discussion by rhkramer** [orbitalquark/textadept#49] [Q&A] [answered]
(I forget how it was phrased, well commented or well documented (or something else), but my questions include:

1) Is the scintilla code part of the 2000 lines of well documented / well commented code?
2) if so, are there more comments / documentation than I'd find on the scintilla site?

Background: I'm not a c/c++ programmer, and I have a terrible time trying to follow the scintilla code -- I wonder(ed) if it would be any easier for me to download and try to understand the scintilla code in Textadept?

**Comment by orbitalquark** [comment, marked as answer]
No, Textadept's core C code is around 2000 lines. Textadept uses Scintilla's existing code unmodified for the most part, so you won't learn anything more downloading Textadept's source :(

**Comment by rhkramer** [comment]
Ahh, ok -- thanks!
